### ISSUE TITLE
NewsDownloader: crash on Edit news feeds

### ISSUE THREAD

**Issue by hius07** [koreader/koreader#9690]
2022.10
```
10/26/22-19:04:20 WARN  error in wrapped function: plugins/newsdownloader.koplugin/main.lua:681: attempt to get length of field 'kv' (a nil value)
stack traceback:
	plugins/newsdownloader.koplugin/main.lua:681: in function 'viewFeedList'
	plugins/newsdownloader.koplugin/main.lua:118: in function <plugins/newsdownloader.koplugin/main.lua:117>
	[C]: in function 'xpcall'
	frontend/ui/trapper.lua:48: in function <frontend/ui/trapper.lua:45> 
```
Reported in https://www.mobileread.com/forums/showthread.php?t=349883.
@roygbyte

**Comment by roygbyte** [comment]
Great! Will look at this tomorrow..

**Comment by poire-z** [comment]
Fixed by #9697.
